make/autoconf/configure.ac

```diff
@@ -194,6 +194,9 @@ TOOLCHAIN_POST_DETECTION
 TOOLCHAIN_SETUP_BUILD_COMPILERS
 TOOLCHAIN_MISC_CHECKS
 
+# Ensure _FILE_OFFSET_BITS is set to 64 for glibc
+AC_SYS_LARGEFILE
```

Comment:
We do not use any of the AC_SYS macros in the JDK build. If you want to add `-D_FILE_OFFSET_BITS=64` to the flags, then you need to do so explicitly, at the right location. And you need to decide if it should be added only for Hotspot, only for the JDK native libraries, or both.